platformio.ini

```diff
@@ -0,0 +1,23 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:esp32]
+platform = espressif32
+board = esp32-s3-devkitc-1
+framework = arduino
+
+build_flags =   
+     -DARDUINO_USB_MODE=1
+     -DARDUINO_USB_CDC_ON_BOOT=1
+     -I src/lib-y-badge
```

Comment:
Sorry, I missed one more line. You shouldn't seed `-I src/lib-y-badge` either.

Reply:
fixed